backstory/pom.xml

```diff
@@ -0,0 +1,82 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.sps</groupId>
+  <artifactId>backstory</artifactId>
```

Comment:
Seems like most of this file should have been unchanged, except fo this line?

Reply:
I agree. It seems like GitHub is struggling to merge them properly. The content is the same. I'll just have to resolve it in merge conflicts.

Reply:
Done

Reply:
Looks like this still needs a merge?

Reply:
I merged, so if this is still showing as new, I'm not sure if there is a way to resolve it. I'm happy to do whatever you think is best though.

Reply:
Thanks the change looks expected now!